views/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "license": "GPL-2.0",
   "dependencies": {
-    "@oat-sa/tao-test-runner": "0.8.2",
-    "@oat-sa/tao-test-runner-qti": "2.30.3"
+    "@oat-sa/tao-test-runner": "^0.9.0",
```

Comment:
As I see, we used exact version in the package.json. Maybe we should continue, even if I don't understand why we did it.